kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -224,8 +225,13 @@ def check_stage_counts(self, stage):
             extract_disc = self.stages[ExtractStage].concept_discovery_dict
             if extract_disc and extract_disc.get('sources'):
                 passed, message = stage_analyses.compare_counts(
-                    discovery_sources,
-                    extract_disc.get('sources')
+                    ExtractStage.__name__,
+                    extract_disc['sources'],
+                    TransformStage.__name__,
+                    {
+                        k: v for k, v in discovery_sources.items()
+                        if UNIQUE_ID_ATTR not in k
+                    }
                 )
                 passed_all = passed_all and passed
                 stage.logger.info(message)
```

Comment:
I think the compare_counts output is too much for an info log level. This type of detailed output should probably be logged at the debug level, and only summaries should be logged at info level. So log this at debug level: 
```
Column CONCEPT|PARTICIPANT|CONSENT_TYPE not equal
ExtractStage {'GRU', 'HMB-IRB', 'DS-OC-PUB-MDS'}
vs
TransformStage {'GRU', None, 'HMB-IRB', 'DS-OC-PUB-MDS'}
Difference is:
{None}
```
And maybe this at info level: 
```
❌ Column values for CONCEPT|PARTICIPANT|CONSENT_TYPE not equal between ExtractStage and TransformStage
# of different values = 1
```

Reply:
√

Reply:
I'm not sure about this. The info messages become not useful anymore by doing that.